### DISCUSSION TITLE
Snapclient configuration issue ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#564] [Fixed Issues]
RPi3A, Raudio i2021015 base 64bit

I have configured one of my other RPI's as a Snapserver - no apparent issues here.  I then configured this RPi as the Snapclient  using the default 800ms latency. When I try to connect to the Snapserver via the Networks submenu icon I get:

![image](https://user-images.githubusercontent.com/25418025/137709645-6a5d296f-eae0-4f4f-abe8-8bd3cce9edfa.png)

I have had to do a web capture from MS Edge as when I try to copy the error text it disappears before I can actually copy it via Ctrl-C. The error (not clear captured in image) is:

Unexpected string in JSON error at position 519

Just see on the Snapserver RPi:

# pacman -Q snapcast; systemctl status snapserver

snapcast 0.24.0-1
● snapserver.service - Snapcast server
     Loaded: loaded (/usr/lib/systemd/system/snapserver.service; enabled; vendor preset: disabled)
     Active: active (running) since Mon 2021-10-18 10:31:23 BST; 47min ago
       Docs: man:snapserver(1)
   Main PID: 938 (snapserver)
      Tasks: 5 (limit: 2138)
     CGroup: /system.slice/snapserver.service
             └─938 /usr/bin/snapserver --logging.sink=system --server.datadir=/var/lib/snapserver

Oct 18 10:31:23 RAOffice snapserver[938]: Exception: end of file
Oct 18 10:31:24 RAOffice snapserver[938]: Service 'Snapcast' successfully established.
Oct 18 10:31:58 RAOffice snapserver[938]: StreamServer::NewConnection: 192.168.86.35
Oct 18 10:31:58 RAOffice snapserver[938]: Hello from b8:27:eb:58:31:79, host: rAudio-1, v0.25.0, ClientName: Snapclient, OS: Arch Linux ARM, Arch: aarch64, Protocol version: 2
Oct 18 10:44:14 RAOffice snapserver[938]: Error reading message header of length 0: End of file
Oct 18 10:44:14 RAOffice snapserver[938]: onDisconnect: b8:27:eb:58:31:79
Oct 18 10:46:41 RAOffice snapserver[938]: StreamServer::NewConnection: 192.168.86.35
Oct 18 10:46:41 RAOffice snapserver[938]: Hello from b8:27:eb:58:31:79, host: rAudio-1, v0.25.0, ClientName: Snapclient, OS: Arch Linux ARM, Arch: aarch64, Protocol version: 2
Oct 18 11:02:42 RAOffice snapserver[938]: Error reading message header of length 0: End of file
Oct 18 11:02:42 RAOffice snapserver[938]: onDisconnect: b8:27:eb:58:31:79

192.168.86.35 is the RPi where the Snapclient error occurs.

Chris

**Comment by rern** [comment]
Fixed.

**Comment by ceejayemm** [comment]
@rern 
I updated to 2021114 on both the SnapServer RA and SnapClient RA machines and while I no longer get the error reported above I do not get any sound output from the SnapClient machine when using SnapClient.  I get the artwork, time etc being displayed on the client Playback screen but no sound output.  I can hear sound on the SnapServer machine. Any items in the local Library on the SnapClient machine play normally with sound when SnapClient is exited.

Thanks

Chris
